startup/01_set_simplified_environment.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
```

Comment:
Did you want this permission to change?

Reply:
I was wrong about this. These are executables, so they need to have 755.